lib/context.js

```diff
@@ -105,24 +106,19 @@ api.process = ({activeCtx, localCtx, options}) => {
     if('@base' in ctx) {
       let base = ctx['@base'];
 
-      // clear base
       if(base === null) {
-        base = null;
-      } else if(!_isString(base)) {
-        throw new JsonLdError(
-          'Invalid JSON-LD syntax; the value of "@base" in a ' +
-          '@context must be a string or null.',
-          'jsonld.SyntaxError', {code: 'invalid base IRI', context: ctx});
-      } else if(base !== '' && !_isAbsoluteIri(base)) {
+        // no action
```

Comment:
Testing this would seem to do nothing for any relative (non-absolute) IRI which isn't the empty string, and not execute the code at 114 below.

Reply:
@davidlehn common on this?

Reply:
I don't understand the issue.  It does nothing for null, else do something if absolute, else do something if relative (which is a lame check so will just run on any string), else fail (ie, non string or something).

Reply:
This would seem to be true of `base = 'a/b'`, as `base !== '' && !_isAbsoluteIri(base)`. This would seem to be where 114 comes in `base = parseUrl(prependBase(activeCtx['@base'].href, base))`.

Reply:
I'm still confused.  Is that line 114 wrong?  I guessed what to do there.  The code previously didn't support relative IRIs at all.  What's an example that would fail?

Reply:
No, you're right, I was getting confused by old vs new code. This should be good.